pages/common/pnpm.md

```diff
@@ -14,19 +14,19 @@
 
 - Download a specific version of a package and add it to the list of dependencies in `package.json`:
 
-`pnpm install {{module_name}}@{{version}}`
+`pnpm add {{module_name}}@{{version}}`
 
-- Download a package and add it to the list of dev dependencies in `package.json`:
+- Download a package and add it to the list of [D]ev dependencies in `package.json`:
```

Comment:
I'm aware that long parameters are encouraged. But I doubt anyone would prefer to type `--save-dev` than `-D`.